CDP4DalJsonSerializer/JsonConverter/PostOperationJsonConverter.cs

```diff
@@ -0,0 +1,151 @@
+﻿// -------------------------------------------------------------------------------------------------------------------------------
+// <copyright file="PostOperationJsonConverter.cs" company="RHEA System S.A.">
+//    Copyright (c) 2015-2024 RHEA System S.A.
+// 
+//    Authors: Sam Gerené, Alex Vorobiev, Alexander van Delft, Nathanael Smiechowski, Antoine Théate, Omar Elebiary, Jaime Bernar
+// 
+//    This file is part of CDP4-COMET SDK Community Edition
+// 
+//    The CDP4-COMET SDK Community Edition is free software; you can redistribute it and/or
+//    modify it under the terms of the GNU Lesser General Public
+//    License as published by the Free Software Foundation; either
+//    version 3 of the License, or (at your option) any later version.
+// 
+//    The CDP4-COMET SDK Community Edition is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+//    Lesser General Public License for more details.
+// 
+//    You should have received a copy of the GNU Lesser General Public License
+//    along with this program; if not, write to the Free Software Foundation,
+//    Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+// </copyright>
+// -------------------------------------------------------------------------------------------------------------------------------
+
+namespace CDP4DalJsonSerializer.JsonConverter
+{
+    using System;
+    using System.IO;
+    using System.Runtime.Serialization;
+    using System.Text.Json;
+    using System.Text.Json.Serialization;
+
+    using CDP4Common.Polyfills;
+
+    using CDP4DalCommon.Protocol.Operations;
+
+    using CDP4DalJsonSerializer.Extensions;
+
+    using NLog;
+
+    /// <summary>
+    /// The <see cref="JsonConverter" /> for <see cref="PostOperation" />s
+    /// </summary>
+    public class PostOperationJsonConverter: JsonConverter<PostOperation>
+    {
+        /// <summary>
+        /// The NLog logger
+        /// </summary>
+        private static readonly Logger Logger = LogManager.GetCurrentClassLogger();
+
+        /// <summary>
+        /// Asserts that the <see cref="PostOperation.Copy"/> have to be ignored
+        /// </summary>
+        private readonly bool ignoreCopyProperty;
+
+        /// <summary>
+        /// Initializes a new <see cref="PostOperationJsonConverter" /> instance.
+        /// </summary>
+        /// <param name="ignoreCopyProperty">Asserts that the <see cref="PostOperation.Copy"/> have to be ignored</param>
+        public PostOperationJsonConverter(bool ignoreCopyProperty = false)
```

Comment:
the default should be true, since this is special for COMET and not supported by version 1,0.0 of 10-25